src/loading/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import injectSheet from 'react-jss';
+import { compose } from 'recompose';
+import classNames from 'classnames';
+import * as colors from '../theme/colors';
+
+const styles = {
+  '@keyframes placeHolderShimmer': {
+    '0%': {backgroundColor: '#dddddd'},
+    '35%': {backgroundColor: '#dddddd'},
+    '50%': {backgroundColor: '#eeeeee'},
+    '65%': {backgroundColor: '#dddddd'},
+    '100%': {backgroundColor: '#dddddd'},
+  },
+  shimmer: {
+    animationDuration: '1.5s',
+    animationIterationCount: 'infinite',
+    animationName: 'placeHolderShimmer',
+    animationTimingFunction: 'linear',
+    color: 'rgba(0,0,0,0)',
+    textDecoration: 'none',
+    borderRadius: '4px',
+    height: 50,
+    width: 200,
```

Comment:
If it's up to the user to define a `width` and `height` inline on the component, is there any point in setting here?

Would it be better to allow the component to accept `children` so that the component can wrap that in a shimmer? Then, it is up to the user to set the size of the child, and by extension the placeholder.